service-utils/gen-ssl.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+path_to_etc=$1
+openssl_conf_path=$2
+
+echo "Setting up credential directory structure..."
+mkdir -p $path_to_etc/creds/root/certs
+mkdir -p $path_to_etc/creds/root/keys
+mkdir -p $path_to_etc/creds/services/certs
+mkdir -p $path_to_etc/creds/services/keys
+
+mkdir tmp && cd tmp
+
+echo "Generating root certs..."
+openssl genrsa -passout pass:1111 -des3 -out ca.key.pem 4096
+
+openssl req -passin pass:1111 -new -x509 -days 7300 -key ca.key.pem -out ca.cert.pem -subj  "/C=FR/ST=Paris/L=Paris/O=Test/OU=Test/CN=root"
+
+echo "Generating intermediate certs..."
+openssl genrsa -passout pass:1111 -des3 -out ca-intermediate.key.pem 4096
+
+openssl req -passin pass:1111 -new -key ca-intermediate.key.pem -out ca-intermediate.csr -subj  "/C=FR/ST=Paris/L=Paris/O=Test/OU=Test/CN=intermediate"
+
+openssl x509 -req -passin pass:1111 -days 365 -extensions v3_intermediate_ca -extfile $openssl_conf_path -in ca-intermediate.csr -CA ca.cert.pem -CAkey ca.key.pem -set_serial 01 -out ca-intermediate.cert.pem
```

Comment:
$openssl_conf path needs to be appropriately handled to use /etc/ssl/openssl.ct.conf or something. May be in a next PR.